t/variation.t

```diff
@@ -29,6 +29,7 @@ use Test::Differences;
 use Catalyst::Test ();
 use Bio::EnsEMBL::Test::MultiTestDB;
 use Bio::EnsEMBL::Test::TestUtils;
+use Storable qw(dclone);
```

Comment:
is Storable already part of the normal packages? otherwise it should be added to the cpanfile as well